pkgs/development/cuda-modules/buildRedist/default.nix

```diff
@@ -332,6 +341,16 @@ extendMkDerivation {
 
       inherit doInstallCheck;
       inherit allowFHSReferences;
+      inherit includeRemoveStubsFromRunpathHook;
+
+      postFixup = postFixup + ''
+        if [[ -n "''${includeRemoveStubsFromRunpathHook:-}" ]] ; then
+          nixLog "installing stub removal runpath hook"
+          mkdir -p "''${!outputStubs:?}/nix-support"
+          printWords >>"''${!outputStubs:?}/nix-support/propagated-build-inputs" \
+            "${getDev removeStubsFromRunpathHook}"
```

Comment:
Ah damn now I understand what you meant by "all packages depending on the hook". I guess more generally we want both setup-time and eval-time, but I feel pure evil to torture you with nits any longer

Reply:
Fixed in 413abbfe75a776b3db48510dab358766486ea800.